src/lib/components/event/event-details-row-expanded.svelte

```diff
@@ -13,6 +13,7 @@
     shouldDisplayAsTaskQueueLink,
```

Comment:
Looks like there's an unused `routeForWorkers` being imported if you feel inclined to remove it 🗑️